utilities/README.md

```diff
@@ -33,7 +33,7 @@ For example, the following command updates all C# projects in the current direct
 > Windows PowerShell comes pre-installed with Windows 10.
 > Download PowerShell Core for Windows, macOS, or Linux at https://github.com/PowerShell/PowerShell.
 
-This PowerShell module provides functionality for invoking [NWChem](http://nwchem-sw.org) using [Docker](https://docker.com/), and for using NWChem to produce [Broombridge](https://docs.microsoft.com/azure/quantum/user-guide/libraries/chemistry/schema/broombridge) documents.
+This PowerShell module provides functionality for invoking [NWChem](https://www.nwchem-sw.org/) using [Docker](https://docker.com/), and for using NWChem to produce [Broombridge](https://docs.microsoft.com/azure/quantum/user-guide/libraries/chemistry/schema/broombridge) documents.
```

Comment:
Interesting that this gets flagged. This link didn't used to support HTTPS, but it looks like now the HTTP version redirects to HTTPS? Anyway, agree with the fix in any case.